contrib/fedora/cjdns.genconf.patch

```diff
@@ -0,0 +1,20 @@
+diff -up ./client/cjdroute2.c.genconf ./client/cjdroute2.c
+--- ./client/cjdroute2.c.genconf	2015-09-26 23:08:20.832843690 -0400
++++ ./client/cjdroute2.c	2015-09-26 23:13:05.288687831 -0400
+@@ -306,14 +306,14 @@ static int genconf(struct Random* rand,
+            "        // and ETHInterface will be unable to hot-add new interfaces\n"
+            "        // Use { \"setuser\": 0 } to disable.\n"
+            "        // Default: enabled with keepNetAdmin\n"
+-           "        { \"setuser\": \"nobody\", \"keepNetAdmin\": 1 },\n"
++           "        { \"setuser\": \"cjdns\", \"keepNetAdmin\": 1 },\n"
```

Comment:
This one will also probably break often and people are not going to care to test their pull requests on fedora, better approach would probably be to introduce command line flags   `--setuser` and `--chrootto` so this can be specified in your scripts.